CondCore/ESSources/python/CondDBESSource_cfi.py

```diff
@@ -20,5 +20,6 @@
     RefreshEachRun   = False,
     RefreshOpenIOVs  = False,
     pfnPostfix       = '',
-    pfnPrefix        = '' ,
+    pfnPrefix        = '',
+    recordsToDebug   = [],
```

Comment:
Should this remain empty or are there any records for which this logging should be printed by default?

Reply:
as far as the application is only for HLT and only for the beam spot, I would keep this empty and let the individual clients customize as needed.